web_src/css/repository.css

```
@@ -3485,3 +3485,15 @@ td.blob-excerpt {
 .search-fullname {
   color: var(--color-text-light-2);
 }
+
+.pinned-issue-card {
+  margin: 4px 2px !important;
+  border-radius: 5px !important;
+  width: calc(100% - 4px) !important;
+  padding: 0.5rem !important;
+  min-height: auto !important;
+  max-width: calc(100% * (1/3) - 10px - 1px);
```

Isn't that the same as 
```suggestion
  max-width: calc(33% - 10px - 1px);
```